log.go

```
@@ -50,21 +50,29 @@ func (a AntiLog) Write(msg string, fields ...Field) {
 		PrependUnique(encodeFieldList(fields)).
 		PrependUnique(a.Fields)
 
-	for ix := 0; ix < len(encodedFields); ix++ {
-		field := encodedFields[ix]
-		if field.Key() != "message" || field.Key() != "timestamp" {
+	var sb bytes.Buffer
+	sb.WriteString(`{ "timestamp": "`)
+	sb.WriteString(now.Format(time.RFC3339))
+	sb.WriteString(`", "message": `)
+	json.NewEncoder(&sb).Encode(msg)
+
+	for _, field := range encodedFields {
+		key := field.Key()
+		if key == "message" || key == "timestamp" {
 			continue
 		}
-		encodedFields = append(encodedFields[0:ix], encodedFields[ix+1:len(encodedFields)]...)
+		sb.WriteString(", ")
+		sb.WriteString(key)
+		sb.WriteString(`: `)
+		sb.WriteString(field.Value())
 	}
+	sb.WriteString(` }`)
 
-	encodedFields = append(encodeFieldList([]Field{"timestamp", now.Format(time.RFC3339), "message", msg}), encodedFields...)
-
-	if a.Writer == nil {
-		a.Writer = os.Stderr
+	w := a.Writer
+	if w == nil {
+		w = os.Stderr
 	}
-
-	fmt.Fprintln(a.Writer, encodedFieldsToJSONObject(encodedFields))
+	w.Write(sb.Bytes())
```

You should probably dogsled this, as it has return values. 
```
_, _ = w.Write(...)
```

---

(so it's explicit the error is being ignored)